src/Scripts/Game/Components/Vehicle/EL_LicensePlateManagerComponent.c

```diff
@@ -16,63 +18,34 @@ class EL_LicensePlatePointInfo: PointInfo
 	}
 }
 
-class EL_LicensePlateManagerComponent: ScriptComponent
+class EL_LicensePlateManagerComponent : ScriptComponent
 {
 	[Attribute(uiwidget: UIWidgets.Auto)]
-	ref array<ref EL_LicensePlatePointInfo> m_Plates;
+	ref array<ref EL_LicensePlatePointInfo> m_aPlates;
 	
 	[Attribute("{E95486C43308F36B}Prefabs/Vehicles/LicensePlate/LicensePlate.et")]
 	protected ResourceName m_LicensePlatePrefab;
 	
 	[RplProp(onRplName: "OnRegistrationUpdated")]
-	string m_Registration;
-	
-	override void EOnInit(IEntity owner)
-	{
-		Resource resource = Resource.Load(m_LicensePlatePrefab);
-		
-		if (!resource.IsValid()) return;
-		
-		EntitySpawnParams params();
-		params.Parent = owner;
-		params.TransformMode = ETransformMode.LOCAL;
-		
-		RplComponent rpl = RplComponent.Cast(owner.FindComponent(RplComponent));
-		
-		if (GetGame().InPlayMode() && rpl && rpl.IsMaster())
-		{
-			Resource container = BaseContainerTools.LoadContainer("{B1DD7B5D4812AB19}Configs/Vehicles/VehicleSettings.conf");
-			EL_VehicleSettings vehicleSettings = EL_VehicleSettings.Cast(BaseContainerTools.CreateInstanceFromContainer(container.GetResource().ToBaseContainer()));
-			m_Registration = vehicleSettings.m_LicensePlateGenerator.GenerateLicensePlate();
-		}
-		
-		for (int i = 0; i < m_Plates.Count(); i++)
-		{
-			auto plate = m_Plates[i];
-			GetPositionFromPoint(i, params.Transform);
-
-			plate.m_Object = EL_LicensePlateEntity.Cast(GetGame().SpawnEntityPrefabLocal(resource, owner.GetWorld(), params));
-			Vehicle veh = Vehicle.Cast(owner);
-			
-			veh.AddChild(plate.m_Object, -1, EAddChildFlags.RECALC_LOCAL_TRANSFORM);
-			plate.m_Object.m_LicensePlateManager = this;
-		}
-	}
+	string m_sRegistration;
 	
+	//------------------------------------------------------------------------------------------------
 	void OnRegistrationUpdated()
 	{
-		foreach (auto plate : m_Plates)
+		foreach (auto plate : m_aPlates)
 		{
-			plate.m_Object.m_TextWidget.SetText(m_Registration);
+			plate.m_Object.m_TextWidget.SetText(m_sRegistration);
 		}
 	}
 	
+	//------------------------------------------------------------------------------------------------
 	override void OnPostInit(IEntity owner)
```

Comment:
Not mentioned in the conventions but I think this should be below general methods

Reply:
I though about it too. In my opinion the best placement for this method would be next to init method but that will break the convention. So maybe is should be the first method after general methods. What do you think?

Reply:
Yes after general methods and right over EOnInit

Reply:
Moved method